README.rst

```diff
@@ -41,23 +41,41 @@ Some noteworthy flags::
 
     codespell -w, --write-changes
 
-The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.::
+The ``-w`` flag will actually implement the changes recommended by codespell.
+Not running with ``-w`` flag is the same as with doing a dry run. It is
```

Comment:
I think grammatically it should probably be like this:
```suggestion
Not running with ``-w`` flag is the same as doing a dry run. It is
```

Reply:
This is not something that I have added. Anyway, I agree that "with" should not be here.

I should have changed that description to something more concise and clear. Done.